test/functional/testCases.js

```diff
@@ -2424,6 +2424,7 @@ const testCases = [
             {
                 shouldName: 'C - WHEN sending only b IT should store only [{object_id:b}] attrs into Context Broker',
                 type: 'single',
+                skip: '!lib', // FIXME: https://github.com/telefonicaid/iotagent-json/issues/782
```

Comment:
The usage of `lib` is documented in https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/functional/README.md, but not the usage of `!lib`. Maybe that document should be improved to explain the `!lib` case.

Reply:
You are right. Anyway, I think the best is to create a separated PR.

Reply:
Let's do it and merge both at the same time?

Reply:
https://github.com/telefonicaid/iotagent-node-lib/pull/1532